Sources/Telenor-ID-iOS-SDK/Validation/Validator.swift

```diff
@@ -36,36 +36,16 @@ public enum Validator {
             """)
         }
 
-        guard let audience = token["aud"] as? [String] else {
+        guard let audience = token["aud"] as? String else {
```

Comment:
It was a bug to only support an arrays for the audience claim. It is also a bug only to support a string value. We should therefore have code that supports either an array or a string.